src/libsystemd/libsystemd.sym

```diff
@@ -682,3 +682,14 @@ global:
         sd_bus_object_vtable_format;
         sd_event_source_disable_unref;
 } LIBSYSTEMD_241;
+
+LIBSYSTEMD_245 {
+global:
+        sd_event_add_child_pidfd;
+        sd_event_source_get_child_pidfd;
+        sd_event_source_get_child_pidfd_own;
+        sd_event_source_set_child_pidfd_own;
+        sd_event_source_get_child_process_own;
+        sd_event_source_set_child_process_own;
+        sd_event_source_send_child_signal;
+} LIBSYSTEMD_243;
```

Comment:
Please drop this part for now. Let's let this bake for a bit before releasing. I think the PR could be merged right before release. (If you could e.g. submit just this patch as a seperate "postponed" PR, that'd be great.)

Reply:
As mentioned below, I'd probably prefer delaying the whole PR to 245